NuGet.config

```diff
@@ -6,7 +6,7 @@
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" protocolVersion="3" />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" protocolVersion="3" />
     <add key="dotnet internal feed" value="https://dotnetfeed.blob.core.windows.net/dotnet-core/index.json" protocolVersion="3" />
```

Comment:
you can probably remove this feed too

Reply:
I think it was originally added for this tool:

https://github.com/xamarin/xamarin-android/blob/a22a0368dc791a8fa8d505a8b52c9e1f3906339a/build-tools/Xamarin.Android.Tools.BootstrapTasks/Xamarin.Android.Tools.BootstrapTasks/CheckApiCompatibility.cs#L146

I'll check if it works to remove it, though.

Reply:
It does seem to be in this other feed:
```
OK https://pkgs.dev.azure.com/dnceng/9ee6d478-d288-47f7-aacc-f6e6d082ae6d/_packaging/1a5f89f6-d8da-4080-b15f-242650c914a8/nuget/v3/flat2/microsoft.dotnet.apicompat/5.0.0-beta.20181.7/microsoft.dotnet.apicompat.5.0.0-beta.20181.7.nupkg 261ms
```
I'll remove it.